en/Single-Entrypoint/Single-Entrypoint.md

```diff
@@ -1,3 +1,3 @@
 ---
-name: Single Entrypoint
+title: Single Entrypoint
```

Comment:
for category metadata we use name, you did this correctly in basic-concepts

Reply:
Oh lame, my bad, I have it in my notes as "Name changing to Title."  I'll change it back now.